.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+export default {
```

Comment:
Codacy has a fix for the issue: [Do not use "module".](https://app.codacy.com/gh/devntv/useHooks_Apply_projects/pullRequest?prid=9710929)
```suggestion
export default {
```